integration_tests/features/WalletQuery.feature

```diff
@@ -11,8 +11,23 @@ Feature: Wallet Querying
     # Using GRPC
     Given I have a seed node SeedA
     When I mine a block on SeedA with coinbase CB1
+    When I mine 2 blocks on SeedA
     When I create a transaction TX1 spending CB1 to UTX1
     When I submit transaction TX1 to SeedA
     Then TX1 is in the mempool
     When I mine 1 blocks on SeedA
     Then the UTXO UTX1 has been mined according to SeedA
+
+
+    @critical
+  Scenario: As a wallet I cannot submit a locked coinbase transaction
+    # Using GRPC
+    Given I have a seed node SeedA
+    When I mine a block on SeedA with coinbase CB1
+    When I create a transaction TX1 spending CB1 to UTX1
+    When I submit locked transaction TX1 to SeedA
+    When I mine 2 blocks on SeedA
```

Comment:
There should be a test (Then) before this line to check that the transaction was rejected
e.g.
Then the transaction is rejected